Documentation/install/guides/kops.rst

```diff
@@ -0,0 +1,35 @@
+**********************************
+Kubernetes Kops Installation Guide
+**********************************
+
+As of `kops<https://github.com/kubernetes/kops>`_ 1.9 release, Cilium is a supported CNI plugin for kops-deployed clusters.
+
+Cilium needs a newer kernel version then the default kops images provide (minimum kernel version for Cilium is 4.8), so you need to supply an `ami<https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html>`_ which will be new enough for Cilium to run on it.
```

Comment:
in "a newer kernel version then the default kops", "then" should be "than"

Reply:
done